segmentation-templates-extension/locales/en.default.json.liquid

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "{{ name }}",
```

Comment:
I think we should just merge `title` and `name` to avoid confusion
```suggestion
  "name": "First-time customers who bought specific products",
```